HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 History
 -------
 
+1.2.3 (2017-01-24)
+------------------
+* New decorator to log function calls.
```

Comment:
You should see the commit log for other changes that have been included meanwhile